roles/zammad/templates/zammad.yml.j2

```
@@ -0,0 +1,125 @@
+---
+version: '3.8'
+
+x-shared:
+  zammad-service: &zammad-service
+    environment: &zammad-environment
+      MEMCACHE_SERVERS: zammad-memcached:11211
+      POSTGRESQL_DB: {{ zammad_db_name }}
+      POSTGRESQL_HOST: zammad-postgresql
+      POSTGRESQL_USER: {{ zammad_db_user }}
+      POSTGRESQL_PASS: {{ zammad_db_pass }}
+      POSTGRESQL_PORT: 5432
+      POSTGRESQL_OPTIONS: ?pool=50
+      POSTGRESQL_DB_CREATE:
+      REDIS_URL: redis://zammad-redis:6379
+      # Backup settings
+      BACKUP_DIR: "/var/tmp/zammad"
+      BACKUP_TIME: "03:00"
+      HOLD_DAYS: "7"
+      TZ: "Europe/Berlin"
+      # Allow passing in these variables via .env:
+      AUTOWIZARD_JSON:
+      AUTOWIZARD_RELATIVE_PATH:
+      ELASTICSEARCH_ENABLED:
+      ELASTICSEARCH_HOST:
+      ELASTICSEARCH_PORT:
+      ELASTICSEARCH_SCHEMA:
+      ELASTICSEARCH_NAMESPACE:
+      ELASTICSEARCH_REINDEX:
+      ELASTICSEARCH_SSL_VERIFY:
+      NGINX_PORT:
+      NGINX_SERVER_NAME:
+      NGINX_SERVER_SCHEME:
+      RAILS_TRUSTED_PROXIES:
```

Hier muss vermutlich unser Docker Host rein, oder?
```suggestion
      RAILS_TRUSTED_PROXIES: "['127.0.0.1', '::1', '172.17.0.1']"
```